lib/solidus_extension_dev_tools/rubocop.rb

```diff
@@ -1,3 +1,5 @@
 # frozen_string_literal: true
-# This file is provided to easily provide custom cops at some point in the future without requiring
-# extensions to update their .rubocop.yml.
+
+require File.expand_path('rubocop/inject', __dir__)
+
+SolidusExtensionDevTools::RuboCop::Inject.defaults!
```

Comment:
What about changing this to `SolidusExtensionDevTools::RuboCop.inject_defaults!`

It would simplify thing a bit and it's still very clear.